api/helpers/lock-game.js

```diff
@@ -0,0 +1,63 @@
+const { randomUUID } = require('crypto'); // Added in: node v14.17.0
+const dayjs = require('dayjs');
+
+async function sleep(durationInMillis) {
+  return new Promise((resolve) => {
+    setTimeout(resolve, durationInMillis);
+  });
+}
```

Comment:
A suggestion I got from ButterBothSides was to pull `sleep()` into its own utility function. I'm open to doing so, but my initial instinct is to wait for a second usecase before relocating this so that until then, `sleep()` is colocated with its only caller. Curious for your thoughts @seriouslysean

Reply:
Typically plan for the now, I wouldn’t move it unless you need to use it elsewhere.